infrastructure/provision-rails-assets.yml

```diff
@@ -219,6 +219,29 @@
         insertafter='^http {'
         line='        charset utf-8;'
       notify: restart nginx
+    - name: "add json logging format to nginx"
+      blockinfile:
+      dest: /etc/nginx/nginx.conf
+      insertafter: '^http {'
+      content: |
+        log_format logstash_json '{ "@timestamp": "$time_iso8601", '
```

Comment:
Neat. I had no idea about this. What about json escaping, though? Like if someone puts a `"` in their useragent string?